scripts/src/main/scala/com/gu/mediaservice/scripts/EsScript.scala

```diff
@@ -13,7 +13,7 @@ object MoveIndex extends EsScript {
   def run(esHost: String, extraArgs: List[String]) {
 
     object EsClient extends ElasticSearchClient {
-      val imagesAlias = "imagesAlias"
+      val imagesAlias = "readAlias"
```

Comment:
can this be read from the `Config` object like [here](https://github.com/guardian/grid/pull/1701/files#diff-c8a14be53a6e8680f2e620d6c17edaadR56)